project-addons/custom_account/partner.py

```diff
@@ -115,3 +115,8 @@ def on_change_user_id(self):
         self.payment_responsible_id = self.user_id.id
         if self.user_id and self.user_id.default_section_id:
             self.section_id = self.user_id.default_section_id.id
+
+    @api.multi
+    def action_done(self):
```

Comment:
Hay que llamar a super y luego añadir lo que quieras peor sino se rompe la herencia

Reply:
Pero el super lo que hace es esto: 
![image](https://user-images.githubusercontent.com/26384249/35635503-bf6d2820-06ae-11e8-9146-70b1c1349c52.png)
y es justo lo que quiero evitar, que ponga a False el campo "payment_responsible_id"

Reply:
Ya pero hay formas de hacerlo sin romper la herencia, puedes volver a escribirle el responsable después por ejemplo, o mejor, podrías pasarle algo en el contexto y si ese algo se recibe en el write eliminamos el campo payment_responsible_id del vals por ejemplo. Podemos dejarlo así si queréis pero no es buena práctica, ya podemos estar ocasionando otros problemas sin darnos cuenta, evitado que se ejecute código del módulo ya instalado y que heredaban esta funcionalidad base y desde luego provocamos limiraciones a futuro.

Reply:
He actualizado esta parte con mi último commit. ¿Está mejor ahora con el cambio, no?

Reply:
Sí